fetch-number-of-jobs-from-all-connected-controllers.groovy

```diff
@@ -0,0 +1,27 @@
+import com.cloudbees.opscenter.server.clusterops.steps.MasterGroovyClusterOpStep
+
+def retour = ''
+def stream = new ByteArrayOutputStream();
+def listener = new StreamBuildListener(stream);
+
+Jenkins.instance.getAllItems(com.cloudbees.opscenter.server.model.ConnectedMaster).each{
+    it.channel?.call(new MasterGroovyClusterOpStep.Script("""
+        class composedItem {
+            Object controllerName
+            Integer numberOfJobs
+            String allJobs
+        }
+        
+        jobsController = Jenkins.instance.items.findAll()
```

Comment:
This will find everything. no? Folders, etc, as well.

Reply:
Maybe add example output to the PR

Reply:
I have modified this to use the snippet below instead. This way I will search for all Jobs (without Folders and other auxiliary Items) in the complete instance and not only in the root level as findAll() does. 

`jenkins.model.Jenkins.get().allItems(hudson.model.Job).each {
      jobsController.push(it)
}`

Reply:
I will add the output of the modified script in the comments. Thanks!